docs/phase1-understand.md

```diff
@@ -0,0 +1,31 @@
+---
+id: phase1-understand
+title: Phase 1: Understand
+sidebar_label: Phase 1: Understand
+---
+
+The exercises in this phase help us understand and empathize with the users'
```

Comment:
use headers to differentiate subsections

Reply:
Headers are defined in the title and is intentional.